ansible/configs/ocp-clientvm/destroy_env.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Delete Infrastructure
+  hosts: localhost
+  connection: local
+  gather_facts: False
+  become: no
+  vars_files:
+    - "./env_vars.yml"
+    - "./env_secret_vars.yml"
+  tasks:
+    - name: Destroy cloudformation template
+      cloudformation:
+        stack_name: "{{project_tag}}"
+        state: "absent"
+        region: "{{aws_region}}"
+        disable_rollback: false
+        tags:
+          Stack: "project {{env_type}}-{{ guid }}"
+      tags: [ destroying, destroy_cf_deployment ]
+      register: cloudformation_result
+      until: cloudformation_result|succeeded
+      retries: {{ cloudformation_retries }}
+      delay: 60
+      ignore_errors: yes
+
+    - name: report Cloudformation error
+      fail:
+        msg: "FAIL {{ project_tag }} Destroy Cloudformation"
+      when: not cloudformation_result|succeeded
+      tags: [ destroying, destroy_cf_deployment ]
```

Comment:
here you could just use the common ec2 destroy playbook:

```---
- name: Import default CloudFormation (aws) destroy playbook
  import_playbook: "{{ANSIBLE_REPO_PATH}}/cloud_providers/{{cloud_provider}}_destroy_env.yml"
```

Reply:
Didn't know that existed. That one doesn't work anyway... doesn't work for ocp-workshop either. I'll change it here. If that works maybe we should change ocp-workshop as well.